packages/components/src/internal/app/utils.ts

```diff
@@ -457,17 +476,17 @@ export function getMenuSectionConfigs(
         moduleContext,
         isBioPrimary && isRequestsEnabled(moduleContext) ? 7 : 12
     );
-    const workflowConfig = getWorkflowSectionConfig(appBase);
+    const workflowConfig = isWorkflowEnabled(moduleContext) ? getWorkflowSectionConfig(appBase) : undefined;
```

Comment:
is there a reason why this call to `getWorkflowSectionConfig` and the `isWorkflowEnabled` aren't just used below at line 486 and 487 instead of creating an undefined var and checking that?